client/src/components/GraphViewerComponent/GraphViewerComponentShared.scss

```diff
@@ -66,4 +66,11 @@
     margin-top: -4px;
     color: black;
   }
+  .ms-MessageBar-Relationship{
+    margin-left: -29px;
+    margin-top: 250px;
+  }
+  .div-MessageBar-Relationship{
```

Comment:
Add a space between end of selector and the `{` (same comment ofr above).

Reply:
Done